packages/perseus/src/util/extract-perseus-data.ts

```diff
@@ -536,4 +538,90 @@ function injectWidgets(
     return context;
 }
 
+/* Widgets that have individual answers for the coach report view */
+const INDIVIDUAL_ANSWER_WIDGETS = [
+    "interactive-graph",
+    "categorizer",
+    "grapher",
+];
+
+/* Widgets that are supported for scoring on the coach report view */
+const SUPPORTED_WIDGETS = [
+    "radio",
+    "numeric-input",
+    "input-number",
+    "expression",
+    ...INDIVIDUAL_ANSWER_WIDGETS,
+];
+
+export const getWidgetTypeFromWidgetKey = (widgetKey: string): string => {
+    return widgetKey.split(" ")[0];
+};
+
+/* Verify if the perseus item has supported widgets for scoring on the coach report view. */
+export const isWrongAnswerSupported = (widgetKeys: Array<string>): boolean => {
+    return (
+        widgetKeys.length !== 0 &&
+        widgetKeys.every((widgetKey) =>
+            SUPPORTED_WIDGETS.includes(getWidgetTypeFromWidgetKey(widgetKey)),
+        )
+    );
+};
+
+/* Verify if the widget key has an individual answer for the coach report view  */
+export const shouldHaveIndividualAnswer = (widgetKey: string): boolean => {
+    return INDIVIDUAL_ANSWER_WIDGETS.includes(
+        getWidgetTypeFromWidgetKey(widgetKey),
+    );
+};
+
+/* Returns the answer userInput submission for various supported widgets on the coach report view */
+// TODO (Third): Fix userInput any types to be specific (LEMS-1834)
+export const getAnswerFromUserInput = (widgetType: string, userInput: any) => {
+    switch (widgetType) {
+        case "categorizer":
+            return userInput.values;
+        case "input-number":
+            return userInput.currentValue;
+        case "numeric-input":
+            return userInput.currentValue;
+        case "radio":
+            return userInput.choicesSelected;
+    }
+    return userInput;
+};
+
+/* Returns the correct answer for a given widget key and Perseus Item */
+// TODO (Third): We should fix the resonse type from getWidget to be specific. (LEMS-1835)
+// TODO (Third): We should also consider adding the getOneCorrectAnswerFromRubric method to all widgets. (LEMS-1836)
+export const getCorrectAnswerForWidgetKey = (
+    widgetKey: string,
+    itemData: PerseusItem,
+): string | undefined => {
+    const rubric = itemData.question.widgets[widgetKey].options;
+    const widgetType = getWidgetTypeFromWidgetKey(widgetKey);
+
+    const widget = Widgets.getWidget(widgetType);
+    // @ts-expect-error - TS2339 - Property 'getOneCorrectAnswerFromRubric' does not exist on type 'ComponentType<any>'.
+    if (!widget?.getOneCorrectAnswerFromRubric) {
+        return;
+    }
+    // @ts-expect-error - TS2339 - Property 'getOneCorrectAnswerFromRubric' does not exist on type 'ComponentType<any>'.
+    return widget.getOneCorrectAnswerFromRubric(rubric);
```

Comment:
nit, I think these can be one since `?` returns undefined if the property doesn't exist

```suggestion
    // @ts-expect-error - TS2339 - Property 'getOneCorrectAnswerFromRubric' does not exist on type 'ComponentType<any>'.
    return widget?.getOneCorrectAnswerFromRubric?.(rubric);
```